### DISCUSSION TITLE
View players per distance

### DISCUSSION THREAD

**Discussion by liexes** [diamond-rip/Eden#400] [Ideas]
Could the function of seeing the users in the tab according to a certain range be added? It is that if there are many players on the server and you enter a game, the tab looks ugly and messy, also, the opponent does not look good, it is somewhat tedious, I suggest this feature or if it cannot be added, how can I add it?

**Comment by GoodestEnglish** [comment]
This is not what you should modify. You should use a modern tablist plugin like what lunar/minehq did, and edit the tablist like what you want. But this way of modifying requires knowledge of plugin coding.

**Reply by liexes** [reply]
Do you know of a plugin that can do this?

**Reply by GoodestEnglish** [reply]
Nope. This feature should be custom coded. I've never heard of this feature before.
